tests/figures/chart/radar_chart_plugin.test.ts

```diff
@@ -35,4 +37,64 @@ describe("radar chart", () => {
       stacked: true,
     });
   });
+
+  test("Radar chart legend", () => {
```

Comment:
any way to simplify/factorize those tests ? A 60 lines test is kinda hard to read, even more so when it's repeated 6 times for each chart type

Reply:
Reduced it to 40 lines :) Anyway, as we already have 20+ lines for checking the result, it would be difficult to reduce it more